src/source/feeds.rst

```diff
@@ -31,6 +82,701 @@ XML file.
+cpc           CPC for the given ad in cents              Yes
+totalBudget   total budget for the given ad in cents     No
+dailyBudget   daily budget for the given ad in cents     No
+============= ========================================== ========
+
+The minimum and maximum values for the total budget depend on the category. 
+If the total budget is not returned with the ad, it means there is an unlimited total budget.
+
+When this value of the daily budget is reached the ad will be paused for the rest of the day. 
+The minimum value depends on the category. Maximum value cannot be higher than the total budget.
+
+The minimum and maximum values of the cost per click (cpc) depend on the category.
+
+======= =========================================================================================================
+Example .. code-block:: html 
+
+            <ad:budget>
+                <ad:totalBudget>5000</ad:totalBudget>
+                <ad:dailyBudget>1000</ad:dailyBudget>
+                <ad:cpc>2</ad:cpc>
+            </ad:budget>
+
+======= =========================================================================================================
+
+.. index:: shippingOptions
+.. _feed_ship:
+
+shippingOptions
+"""""""""""""""
+
+Section with shipping options available for a product.
+Options can be defined for single selected type or both.
+
+============= ========================================== ========
+Name          Description                                Required
+============= ========================================== ========
+shippingType  SHIP, PICKUP                               Yes
+cost          cost of shipping in cents                  No
+time          time it takes to deliver the product       No
+location      pick up location of the product            No
+============= ========================================== ========
+
+*SHIP* means the item can be delivered to the buyer in the provided `time` and for the provided `cost`. 
+For shippingType 'SHIP' provide 'cost' in cents and 'time' in days. 'location' is ignored.
+
+*PICKUP* means the item can be picked up at the provided `location`
+For shippingType 'PICKUP' provide 'location'. Both 'cost' and 'time' are ignored.
+
+Whether shipping options are disabled/optional/mandatory for an ad is configured per category, see :ref:`category_config_v2`.
+
+======= ==============================================================
+Example .. code-block:: html 
+
+            <ad:shippingOptions>
+                    <ad:shippingOption>
+                        <ad:shippingType>PICKUP</ad:shippingType>
+                        <ad:location>1097DN</ad:location>
+                    </ad:shippingOption>
+                </ad:shippingOptions>
+
+======= ==============================================================
+
+.. index:: phoneNumber
+.. _feed_phoneNumber:
+
+phoneNumber
+"""""""""""
+
+The phone number of the advertiser as international phone number format, e.g.
```

Comment:
is it a seller, or an advertiser? :) yes, I know, we have proliferated some things too much

Reply:
seller please!

Reply:
ok